.jsdoc-conf.json

```diff
@@ -0,0 +1,17 @@
+{
+    "plugins": ["plugins/markdown"],
+    "recurseDepth": 10,
+    "source": {
+        "includePattern": ".+\\.(gs|js(doc|x)?)$",
+        "excludePattern": "((^|\\/|\\\\)_|(^|\\/)(jsdoc-out/|code-customized\\.gs$))"
+    },
+    "sourceType": "module",
+    "tags": {
+        "allowUnknownTags": true,
+        "dictionaries": ["jsdoc","closure"]
+    },
+    "templates": {
+        "cleverLinks": false,
+        "monospaceLinks": false
+    }
+}
```

Comment:
Given that we set a default output folder name in the `.gitignore` we could add it here as well:
```
    "opts": {
        "destination": "./jsdoc-out/"
    },
```

I don't know if there are any other useful default options we could set here.

Reply:
Nice :-) I hadn't read the docs for the config-file. `jsdoc-out` was already in the `excludePattern` to avoid any unexpected side-effects. Will add it in a minute (and will update the docs as the optflag now won't be needed by default).

Reply:
...I'm still deciding whether to also add `pedantic`, `verbose`, and `recurse` to the config-file (and remove them as manual flags in the docs), because it could be argued either way whether they are sane **default** behaviour. What do you think? If you can't reply at the moment I will leave it as-is, and that can always be done in a later commit/PR.